### DISCUSSION TITLE
How to disable only one preflight normalization

### DISCUSSION THREAD

**Discussion by agm1984** [tailwindlabs/tailwindcss#8504] [Help] [answered]
I'm using PrimeVue for UI components, and it has a button component. When I use it with Tailwind, Tailwind is taking precedence with the button background colour, so my button background is transparent rather than brand primary colour.

The code is simple:
```vue
<Button label="Log In" class="w-full" />
```

But in the browser dev tools, I see this:
<img width="491" alt="image" src="https://user-images.githubusercontent.com/17655219/171688509-167961e1-4ebe-4108-a635-bada62c579bd.png">

I don't know how to "undo" only the background-colour property.

I found a solution where I add this to the tailwind.config.js file:
```javascript
    corePlugins: {
        preflight: false,
    },
```

But that seems too strong of a response, because it makes all my hyperlinks have an underline by default, so I only want to stop tailwind from applying `background-color: transparent` in preflight. Is there a way to do that?

Is there a modification to tailwind.config.js that will just turn off only the transparent background property?

**Comment by agm1984** [comment, marked as answer]
I found the answer here: https://github.com/tailwindlabs/tailwindcss/discussions/5969

My problem was the base utilities were declared after the PrimeVue theme.

I had this:
``` javascript
import 'vue-toastification/dist/index.css';
import 'primevue/resources/primevue.min.css';
import './primevue-theme.css';
import 'primeicons/primeicons.css';
import './index.css';
```

But I changed it to this:
```javascript
import './index.css';
import 'vue-toastification/dist/index.css';
import 'primevue/resources/primevue.min.css';
import './primevue-theme.css';
import 'primeicons/primeicons.css';
```

**Reply by galih56** [reply]
I'm facing the same issue, Preflight also makes my button background transparent. I'm currently using Ant Design UI library. This library doesn't use inline import as you do. Do you have any idea to solve this?

**Reply by sushantkadam15** [reply]
Did you find a workaround?
